src/lib/shaderloader.js

```diff
@@ -6,6 +6,7 @@ function ShaderLoader () {
 
 ShaderLoader.prototype = {
   loadShadersData: function () {
+    return;
```

Comment:
No, the shader repo was deleted and now we're using just `aframe-registry` to store them, but retrieving them from there will be another PR. I just added it to prevent console errors while trying to fetch the json